src/main/java/org/uast/astgen/codegen/java/NodeTypeConstructor.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * MIT License Copyright (c) 2022 unified-ast
+ * https://github.com/unified-ast/ast-generator/blob/master/LICENSE.txt
+ */
+package org.uast.astgen.codegen.java;
+
+import java.util.List;
+import org.uast.astgen.rules.Child;
+import org.uast.astgen.rules.Descriptor;
+import org.uast.astgen.rules.DescriptorAttribute;
+import org.uast.astgen.rules.Node;
+
+/**
+ * Generates type subclass source code for rules that describe nodes.
+ *
+ * @since 1.0
+ */
+final class NodeTypeConstructor extends NodeConstructor {
+    /**
+     * The 'List&lt;String&gt;' type.
```

Comment:
You may use {@code List<String>} to avoid html restrictions.

Reply:
Fixed